CHANGELOG.md

```diff
@@ -9,6 +9,63 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+
+Note: The repository has moved. The new URL is https://github.com/WordPress/WordPress-Coding-Standards.
+The move does not affect the package name for Packagist. This remains the same: `wp-coding-standards/wpcs`.
+
+### Added
+- New `WordPress.DateTime.CurrentTimeTimestamp` sniff to the `WordPress-Core` ruleset, which checks against the use of the WP native `current_time()` function to retrieve a timestamp as this won't be a _real_ timestamp. Includes an auto-fixer.
+- New `WordPress.DateTime.RestrictedFunctions` sniff to the `WordPress-Core` ruleset, which checks for the use of certain date/time related functions. Initially this sniff forbids the use of the PHP native `date_default_timezone_set()` and `date()` functions.
+- New `WordPress.PHP.DisallowShortTernary` sniff to the `WordPress-Core` ruleset, which, as the name implies, disallows the use of short ternaries.
+- New `WordPress.CodeAnalysis.EscapedNotTranslated` sniff to the `WordPress-Extra` ruleset which will warn when a text string is escaped for output, but not being translated, while the arguments passed to the function call give the impression that translation is intended.
+- New `WordPress.NamingConventions.ValidPostTypeSlug` sniff to the `WordPress-Extra` ruleset which will examine calls to `register_post_type()` and throw errors when an invalid post type slug is used.
+- `Generic.Arrays.DisallowShortArraySyntax` to the `WordPress-Core` ruleset.
+- `WordPress.NamingConventions.PrefixAllGlobals`: the `PHP` prefix has been added to the prefix blacklist as it is reserved by PHP itself.
+- The `wp_sanitize_redirect()` function to the `sanitizingFunctions` list used by the `WordPress.Security.NonceVerification`, `WordPress.Security.ValidatedSanitizedInput` and `WordPress.Security.EscapeOutput` sniffs.
+- The `sanitize_key()` and the `highlight_string()` functions to the `escapingFunctions` list used by the `WordPress.Security.EscapeOutput` sniff.
+- The `RECOVERY_MODE_COOKIE` constant to the list of WP Core constants which may be defined by plugins and themes and therefore don't need to be prefixed (`WordPress.NamingConventions.PrefixAllGlobals`).
+- `$content_width`, `$plugin`, `$mu_plugin` and `$network_plugin` to the list of WP globals which is used by both the `WordPress.Variables.GlobalVariables` and the `WordPress.NamingConventions.PrefixAllGlobals` sniffs.
+- `Sniff::is_short_list()` utility method to determine whether a _short array_ open/close token actually represents a PHP 7.1+ short list.
+- `Sniff::find_list_open_close()` utility method to find the opener and closer for `list()` constructs, including short lists.
+- `Sniff::get_list_variables()` utility method which will retrieve an array with the token pointers to the variables which are being assigned to in a `list()` construct. Includes support for short lists.
+- `Sniff::is_function_deprecated()` `static` utility method to determine whether a declared function has been marked as deprecated in the function docblock.
```

Comment:
```suggestion
- `Sniff::is_function_deprecated()` utility method to determine whether a declared function has been marked as deprecated in the function docblock.
```

The three lines above don't highlight themselves as static methods, so there's no reason for this one too.

Reply:
```suggestion
- `Sniff::is_function_deprecated()` `static` utility method to determine whether a declared function has been marked as deprecated in the function DocBlock.
```

A DocBlock is stylised with capital D and capital B: https://docs.phpdoc.org/guides/docblocks.html

Reply:
> The three lines above don't highlight themselves as static methods, so there's no reason for this one too.

The other three methods _aren't_ static, while this one is. That's the point.

Reply:
Can we at least make it 

> static

rather than

> `static`

There's no reason for it to be marked as code in this sentence.

Reply:
Done